### PR TITLE
updated chrome driver path - added in env variables

### PR DESCRIPTION
**Pull request by ratheesh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by sreehari-aot — APPROVED** [review]
LGTM!

**Review by abhilash-aot — APPROVED** [review]

**Comment by sumesh-aot** [comment]
All pytest were failing, please do not merge any PR unless all CI checks are passed as once you merge the PR it builds image to get it deployed to DEV.

**Pull request merged**
